lib/js/components/Modal/Modal.vue

```diff
@@ -73,6 +74,7 @@
 		z-index: 1;
 
 		&__close:only-child {
+			color: $color-neutral-icon-weak;
```

Comment:
are you sure this color should be placed in this attribute?

Reply:
Why not? This selector is for `svg`. You suggest:
```suggestion
		&__close {
			color: $color-neutral-icon-weak;

			&:only-child {
				margin-left: auto;
			}
		}
```

Reply:
yes, if you don't separate these style blocks and put content in `#header` slot, then the close icon would have different color